Library/.rubocop.yml

```diff
@@ -6,6 +6,12 @@ AllCops:
     - '**/Casks/**/*'
     - '**/vendor/**/*'
 
+require:
+  - ./Homebrew/rubocops/bottle_block_cop.rb
```

Comment:
Let's maybe require a since `Homebrew/rubocops.rb` which in turn includes all our custom cops (just this one for now but we'll add more).